packages/x-charts/src/PieChart/PieArcPlot.tsx

```diff
@@ -132,7 +132,9 @@ function PieArcPlot(props: PieArcPlotProps) {
           endAngle={focusedItem.endAngle}
           paddingAngle={focusedItem.paddingAngle}
           innerRadius={focusedItem.innerRadius}
-          color={focusedItem.color}
+          color="transparent"
+          pointerEvents="none"
+          skipInteraction
```

Comment:
> When the pointer is hover a pie slice and press arrow, the item highlight stay with the pointer instead of following the focus.

If I comment this line, it still seems to work:

https://github.com/user-attachments/assets/299fa825-c772-4211-b272-5240bb8f4038

Am I doing something wrong?

Reply:
Ah, I didn't realize the `PieArc` was being exported. 

> The `skipInteraction` is from us.If you don't whant to get the default interaction handler and add an onClick you can use it.

Sure, makes sense.

> In that specific case we miss the scenario where

Yeah, good point.